governator-core/src/main/java/com/netflix/governator/InjectorBuilder.java

```diff
@@ -62,6 +63,23 @@ public static InjectorBuilder fromModules(List<Module> modules) {
         return new InjectorBuilder(Modules.combine(modules));
     }
     
+    public static <T> InjectorBuilder bindInstance(final Object o) {
```

Comment:
that InjectorBuilder method reduces the test class size by about half for this commit.   It saves boilerplate construction of an inner class extending AbstractModule and binding a single instance by type, with every test method.  Given that the purpose of a 'Builder' pattern like InjectorBuilder is to reduce boilerplate construction for the Injector, I think there's merit to a convenience method like  the bindInstance() methods